cms/static/js/views/settings/grading.js

```diff
@@ -298,10 +310,14 @@ var GradingView = ValidatingView.extend({
         this.descendingCutoffs.push({designation: this.GRADES[gradeLength], cutoff: failBarWidth});
         this.descendingCutoffs[gradeLength - 1]['cutoff'] = Math.round(targetWidth);
 
-        var $newGradeBar = this.gradeCutoffTemplate({ descriptor : this.GRADES[gradeLength],
-            width : targetWidth, removable : true });
+        var newGradeString = this.gradeCutoffTemplate({
+            descriptor : this.GRADES[gradeLength],
+            width : targetWidth,
+            contenteditable: true,
+            removable : true }).toString();
         var gradeDom = this.$el.find('.grades');
-        gradeDom.children().last().before($newGradeBar);
+
+        gradeDom.children().last().before(newGradeString);
```

Comment:
1. @andy-armstrong: How stringently do you want to enforce dropping intermediate variables?  I don't have a huge problem with leaving it, because sometimes it is useful when coding to have the additional name or place for breakpoint, etc.   @awaisdar001 - assuming this variable stays, you could rename to `newGradeHtmlString`. 
2. @andy-armstrong: Do we need to add `before`, `after`, etc. as helpers in addition to `HtmlUtils.append`?